foundation_cms/templates/patterns/components/footer.html

```diff
@@ -34,7 +34,15 @@
         </ul>
       </div>
       <div class="site-footer-ns__grid-cell site-footer-ns__cta">
-        <button type="button" class="site-footer-ns__button site-footer-ns__button--cta">Donate now</button>
+        <!-- Custom markup required until behavior implementation finalized -->
+        <a href="#" class="btn-primary">
```

Comment:
Let's utilize `foundation_cms/templates/patterns/components/button.html` to keep this DRY.